src/test/java/org/jabref/logic/importer/fetcher/IacrEprintFetcherTest.java

```diff
@@ -0,0 +1,121 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.util.Optional;
+
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.ImportFormatPreferences;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BiblatexEntryTypes;
+import org.jabref.model.entry.FieldName;
+import org.jabref.testutils.category.FetcherTest;
+
+import org.junit.Before;
```

Comment:
We just started to use JUnit 5 and it would be nice if you could use the new api (some of the other fetcher tests are already migrated).